examples/quickstart/jupyter-notebooks/notebooks/03-query/04-UnionOperations.ipynb

```diff
@@ -0,0 +1,644 @@
+    "\n",
+    "When we want to apply functions to the _entire_ result set like above, such as using it in a sub-query or a `FROM` clause, Druid changes how it plans and executes the query. Instead of a top level `UNION ALL`, it switches to a \"[table level](https://druid.apache.org/docs/26.0.0/querying/sql.html#table-level)\" query.  For example, to put our very first cell results into `__time` order, we might want to use a query that applies `ORDER BY` to a `UNION` of `set1` (San Francisco flights) and `set2` (Atlanta flights), like the following:\n",
+    "\n",
+    "```sql\n",
+    "SELECT \"Origin\",\n",
+    "    \"Tail_Number\",\n",
+    "    \"Flight_Number_Reporting_Airline\"\n",
+    "FROM (\n",
+    "    SELECT * from set1\n",
+    "    UNION ALL\n",
+    "    SELECT * from set2\n",
+    "    )\n",
+    "ORDER BY __time\n",
+    "```\n",
+    "\n",
+    "Run the cell below to see what happens."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0b557a62-623d-4d16-b4a1-3bd6484efe35",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql = '''\n",
+    "WITH\n",
+    "set1 AS (\n",
+    "  SELECT\n",
+    "  __time,\n",
+    "  \"Origin\",\n",
+    "  \"Tail_Number\",\n",
+    "  \"Flight_Number_Reporting_Airline\"\n",
+    "  FROM \"On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2005_11\"\n",
+    "  WHERE Origin = 'SFO'\n",
+    "  AND DATE_TRUNC('HOUR', __time) = TIMESTAMP '2005-11-01 11:00:00'\n",
+    "  ),\n",
+    "set2 AS (\n",
+    "SELECT\n",
+    "  __time,\n",
+    "  \"Origin\",\n",
+    "  \"Tail_Number\",\n",
+    "  \"Flight_Number_Reporting_Airline\"\n",
+    "  FROM \"On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2005_11\"\n",
+    "  WHERE Origin = 'ATL'\n",
+    "  AND DATE_TRUNC('HOUR', __time) = TIMESTAMP '2005-11-01 11:00:00'\n",
+    "  )\n",
+    "\n",
+    "SELECT __time,\n",
+    "    \"Origin\",\n",
+    "    \"Tail_Number\",\n",
+    "    \"Flight_Number_Reporting_Airline\"\n",
+    "FROM (\n",
+    "    SELECT * from set1\n",
+    "    UNION ALL\n",
+    "    SELECT * from set2\n",
+    "    )\n",
+    "ORDER BY __time\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "25001794-e1a7-4325-adb3-2b8f26036261",
+   "metadata": {},
+   "source": [
+    "Notice the `ClientError` that Druid returns. Rather than simply concatenating two `TABLE`s of results together, Druid switched to table level `UNION ALL`, and with it a switch to [`UNION`](https://druid.apache.org/docs/26.0.0/querying/datasource.html#union) datasources. These cannot be used in the same way as `TABLE`s; there are constraints on what can be done with them. An `EXPLAIN PLAN` for these types of queries shows this in detail:\n",
```

Comment:
```suggestion
    "Notice the `ClientError` that Druid returns. Rather than simply concatenating two `TABLE`s of results together, Druid switched to table level `UNION ALL`. The native query translation uses [`UNION`](https://druid.apache.org/docs/26.0.0/querying/datasource.html#union) datasources. These cannot be used in the same way as `TABLE` datasources; there are constraints on what can be done with them. An `EXPLAIN PLAN` for these types of queries shows this in detail:\n",
```